src/java.base/share/classes/java/net/Inet6Address.java

```diff
@@ -262,10 +262,9 @@ String getHostAddress() {
         }
 
         public boolean equals(Object o) {
-            if (! (o instanceof Inet6AddressHolder)) {
+            if (! (o instanceof Inet6AddressHolder that)) {
```

Comment:
There's a spurious space after "!" can be probably be fixed while you are on that line.

Reply:
Can do. I've removed the space in commit eca9095